src/main/java/com/online/bookstore/model/enumeration/RoleName.java

```diff
@@ -0,0 +1,5 @@
+package com.online.bookstore.model.enumeration;
+
+public enum RoleName {
+    USER, ADMIN
```

Comment:
```suggestion
    USER, 
    ADMIN
```

Reply:
This case is valid for enum, especially when you have a lot of enums

Reply:
> This case is valid for enum, especially when you have a lot of enums

rewritten, thank you